lua/firvish/job_control.lua

```diff
@@ -345,25 +353,20 @@ M.start_job = function(opts)
         api.nvim_buf_set_var(bufnr, "firvish_job_id", job_id)
     end
 
-    s_jobs[job_id] = {
-        bufnr = bufnr,
-        cmd = opts.cmd,
-        title = buf_title,
-        stdout = {},
-        stderr = {},
-        output = {},
-        running = true,
-        is_background_job = opts.is_background_job,
-        start_time = fn.strftime('%H:%M:%S'),
-        finish_time = "",
-        exit_code = nil,
-        is_listed = opts.listed,
-        output_qf = opts.output_qf,
-        output_lqf = opts.output_lqf,
-        bufnr = api.nvim_get_current_buf(),
-        efm = opts.efm,
-        notify = opts.notify
+    local job_info = {
+      bufnr = bufnr,
+      title = buf_title,
+      stdout = {},
+      stderr = {},
+      output = {},
+      running = true,
+      start_time = fn.strftime('%H:%M:%S'),
+      finish_time = "",
+      exit_code = nil,
+      -- FIX: Duplicate.
```

Comment:
Did you intend to leave this in?

Reply:
Ah yes. It was this way before. I had meant to ask you which of the two you thought was correct. I *assume* you want to keep L367 over L357

Reply:
Ah, sorry! It has been a while since I last looked at the code and I hadn't realize that. Since it's not related to the PR, let's just leave it and I'll fix it. Thank you.

Reply:
HA I just fixed it. I can revert if you want

Reply:
No need. All good!